qemu/tests/cfg/serial_modprobe.cfg

```diff
@@ -0,0 +1,14 @@
+- serial_modprobe: install setup image_copy unattended_install.cdrom
+    type = virt_subtest_combine
+    kill_vm_on_error = yes
+    virtio_ports_fixed = "vs1"
+    vrtio_port_type_fixed = "serialport"
+    only Linux
+    login_timeout = 240
+    subtests = check_device remove_mod check_no_mod check_no_device load_mod check_device
+    cmd_check_device = ls /dev/vport0p1
```

Comment:
Hi ypu,
Thank you for your reply
Didn't get the output of "ls /dev/vport0p1"
in my mind, the test will be more clear if we can provide this cmd in .info file, as it's flushed by debug message in debug file.